src/Step/SkipStep.php

```diff
@@ -39,9 +39,14 @@ public function __construct(LoggerInterface $logger, string $stepName)
      * Logs the information about step skipping.
      *
      * @return void
+     * @throws StepException
```

Comment:
these two lines can be changed to `{@inheritDoc}`

Reply:
done